assets/model_monitoring/components/src/shared_utilities/store_url.py

```diff
@@ -254,6 +256,17 @@ def _read_local_file_content(self, relative_path: str = None) -> str:
         with open(full_path) as f:
             return f.read()
 
+    def _create_file_and_append_content(
```

Comment:
we don't need this method if the refactored code works

Reply:
See other comment.